src/main/scala/com/fortysevendeg/scala/android/modules/validators/DummyFormValidation.scala

```diff
@@ -0,0 +1,34 @@
+package com.fortysevendeg.scala.android.modules.validators
+
+import com.fortysevendeg.scala.android.modules.utils.validation.Validator
+import com.fortysevendeg.scala.android.ui.validateforms.FormModel
+
+import scalaz.Validation.FlatMap._
+
+trait DummyFormValidation extends BaseValidation {
+
+  val validationDescriptionAction = "saving form"
+
+  val nameField = "name"
+
+  val emailField = "email"
+
+  val ageField = "age"
+
+  def isValidNameField: Validator[FormModel] =
+    input => hasRequiredField(input.name, nameField, input.name.getStringValue, validationDescriptionAction)(input)
+
+  def isValidEmailField: Validator[FormModel] =
+    input =>
+      for {
+        isFilled <- hasRequiredField(input.email, emailField, input.email.getStringValue, validationDescriptionAction)(input)
```

Comment:
We should perform validation via Applicative and reducing folding over to show all errors.